_pages/en/product-fw-guidance.md

```diff
@@ -0,0 +1,352 @@
+---
+layout: default
+title: Pilot IITB Guidance on Product Management
+ref: product-fw-guidance
+lang: en
+sections: Supporting Documents
+date: "2022-02-04"
+permalink: /product-fw-guidance.html
+---
+<!-- markdownlint-disable MD033 -->
+
+<div class="alert alert-warning">
+    <p align="center"><strong>DISCLAIMER</strong></p>
+    <p>
+This document <strong>does NOT</strong> attempt to explain all aspects of Product Management (such as vision setting, user research, and the different phases a product goes through in its life cycle).
+These are well documented already, such as on <a href="https://www.pmi.org/disciplined-agile/process/product-management">PMI.org</a> and <a href="https://community.pdma.org/knowledgehub/home">PDMA.org</a>.
+    </p>
+    <p>
+    Instead, this documents seeks to provide structure to ESDC Programs and IITB in the <strong>transition</strong> from project to product, <strong>within a government context</strong>.
+    In particular, the rationale and methods to <strong>cross-functional and cross-branch product teams</strong>.
+    </p>
+</div>
+
+<!-- markdownlint-disable MD001 -->
+##### Table of Content <!-- omit in toc -->
+<!-- markdownlint-enable MD001 -->
+- [Introduction](#introduction)
+  - [Document Purpose](#document-purpose)
+  - [References](#references)
+  - [Enquiries](#enquiries)
+- [Narrative](#narrative)
+- [Guidance on Specific Directive Elements](#guidance-on-specific-directive-elements)
+  - [Product Team Setup](#product-team-setup)
+  - [Roles and Responsibilities](#roles-and-responsibilities)
+  - [Oversight and Reporting](#oversight-and-reporting)
+  - [Consequences of Non-Compliance](#consequences-of-non-compliance)
+- [Appendix A - Business View of Job Bank's Family of Products](#appendix-a---business-view-of-job-banks-family-of-products)
+- [Appendix B - IT View of Job Bank's Family of Products of Products](#appendix-b---it-view-of-job-banks-family-of-products-of-products)
+
+## Introduction
+
+### Document Purpose
+
+To provide guidance in implementing the [PILOT IITB Directive on Product Management](product-fw-directive.html) and its associated [Standard](product-fw-standard.html).
+
+The Directive and Standard strive to be succinct. Any context, explications, and situational awareness for the reader has been moved to this document.
+
+### References
+
+This document provides guidance on the following documents:
+
+- **[Pilot IITB Directive on Product Management](product-fw-directive.html)**
+- **[Pilot IITB Standards on Product Management](product-fw-standard.html)**
+
+### Enquiries
+
+Please direct enquiries about this Pilot directive to the IITB IT Strategy team.

```

Comment:
Or if we don't want to link to the generic mailbox (since I'm not sure we are good at checking it), perhaps a link to the 'About Us' page?  @remyb2canada you might want to add Product Management to your interests on the team page. :)